saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -134,6 +135,11 @@ class Meta:
         description = 'Updates a draft order.'
         model = models.Order
 
+    @classmethod
+    def success_response(cls, instance):
+        validate_shipping_method(instance)
```

Comment:
I'm not sure if that's the right place, I feel like we could override the `save()` method, call it with `super()` and then call this function, apart from that this pull requests looks good to me and preferably would like to merge it soon

Reply:
I am with you on that.